### PR TITLE
support html in chatbot

### PR DESCRIPTION
**Pull request by pngwn**
## Description

Please include a concise summary, in clear English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #(issue)

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: gradio/components/chatbot.py [hunk 1]
```diff
@@ -185,10 +187,17 @@ def _preprocess_chat_messages(
         elif isinstance(chat_message, str):
             return chat_message
         elif isinstance(chat_message, ComponentMessage):
-            component = import_component_and_data(chat_message.component.capitalize())
+            capitalized_component = (
+                chat_message.component.upper()
+                if chat_message.component in ("json", "html")
+                else chat_message.component.capitalize()
+            )
+            component = import_component_and_data(capitalized_component)
```
Comment (pngwn) [inline gradio/components/chatbot.py:195]: To handle. `gr.HTML` and `gr.JSON` which don't follow the same pattern as others.

File: gradio/components/chatbot.py [hunk 2]
```diff
@@ -185,10 +187,17 @@ def _preprocess_chat_messages(
...
             if component is not None:
                 instance = component()  # type: ignore
-                if issubclass(instance.data_model, GradioModel):
+                if not instance.data_model:
+                    payload = chat_message.value
+                elif issubclass(instance.data_model, GradioModel):
```
Comment (pngwn) [inline gradio/components/chatbot.py:200]: `issubclass` requires the first arg to be a class and if there is no `data_model` then this isn't the case, so handled this earlier.

File: gradio/components/chatbot.py (hunk 1 above)
Comment (pngwn) [reply gradio/components/chatbot.py:195]: We should probably think of a better way to handle this at some point.

**Review by aliabid94 — APPROVED** [review]
looks great, thanks pete!

**Pull request merged**